### ISSUE TITLE
Minimap empties sometimes

### ISSUE THREAD

**Issue by Zoxjib** [bolrog/d2dx#138]
It happens rather rarely but sometimes a known map disappears. For example farming Tower 1-5 repeatedly and once in 30-50 runs Tower 4 appears unknown as if it's the first time I enter it.

Footage example https://clips.twitch.tv/ShortOptimisticTardigradeSuperVinlin-AWTcusLaR_R1IHa8

**Comment by TheLastPlatypus** [comment]
Have you played in TCP/IP multiplayer in-between?

**Comment by Zoxjib** [comment]
Nope, single-player only (how do you explain Tower 4 disappeared while Tower 5 already known?)
